TestCases.md

```diff
@@ -0,0 +1,224 @@
+4. Ensure the 4 missing words are listed above the grid for drag-and-drop.
+5. Verify the presence of a "Back" button.
+6. Verify the presence of a "Next Step" button.
+7. Verify that the 4 missing words can be dragged and dropped into the empty slots on the grid.
+8. Ensure that each word can only be placed in one slot.
+9. Ensure that only one word can be placed in one slot at a time. If there was already a word, it should pop out to the initial top position above the grid.
+10. Check that a word, once placed, remains in the slot unless dragged out again.
+11. Ensure the user can easily drag the word out of the incorrect slot and try again.
+12. Verify that correctly placing the words into their slots provides visual feedback.
+13. Ensure that an error message or visual indication is provided if the words are placed in the wrong slots.
+
+**Back button**
+
+1. Verify that clicking "Back" navigates the user back to the mnemonic phrase page.
+2. Ensure that the mnemonic phrase is retained when returning to the mnemonic phrase page.
+3. Ensure that clicking 'Back' clears all filled words and prompts the user to fill in the 4 words again.
+
+**Next step button**
+
+1. Verify that clicking "Next Step" is only enabled when all 4 words are correctly placed in the grid.
+2. Ensure that clicking "Next Step" navigates the user to the next page i.e. password setup.
+3. Check behavior when attempting to click "Next Step" with only some of the words correctly placed.
+4. Ensure that clicking 'Next Step' clears all filled words and prompts the user to fill in the 4 words again.
+
+## Password Setup Page
+
+1. Verify the password setup page loads successfully.
+2. Ensure the "Set the password:" input field is visible and accepts input.
+3. Ensure the "Confirm the password:" input field is visible and accepts input.
+4. Verify that by default input is hidden and replaced by "*".
+5. Check that the page displays eye icon buttons for both input fields.
+6. Check that both input fields have the placeholder text "Enter password".
+7. Check that the page displays: "Back" button.
+8. Check that the page displays: Green "Create Wallet" button.
+9. Verify clicking the eye icon reveals the entered password in the "Set the password:" field.
+10. Verify clicking the eye icon reveals the entered password in the "Confirm the password:" field.
+11. Ensure clicking the eye icon again hides the password.
+12. Verify that the "Confirm the password:" field must match the "Set the password:" field.
+13. Verify an appropriate error message is shown if the passwords in the "Set the password" and "Confirm the password" fields do not match.
+14. Verify the system correctly handles and accepts special characters in the password fields.
+15. Verify that the password is securely transmitted and not exposed in any way.
+16. Ensure the app correctly handles password encryption and validation.
```

Comment:
17. Ensure that appropriate error messages is shown when a password doesn't meet the requirements (now it is "Password should contain symbols in upper and lower cases, numbers, and special characters"). 
Ideally to add 18. Ensure that when the focus is on the field the suggestion pop-up is shown - what should password contain